server/api/controllers/webhook/getData.ts

```diff
@@ -0,0 +1,25 @@
+import { firestore } from './firebase';
+import { collection, query, getDocs } from '@firebase/firestore';
+const ref = collection(firestore, 'clients');
+type Record = { board_ID: string; username: string; api_key: string };
```

Comment:
Type is defined twice. Define it once and import it.

Reply:
So moving type Record = { board_ID: string; username: string; api_key: string }; this line to a new file and importing it.